go/internal/initutil/ipfs.go

```diff
@@ -181,8 +182,58 @@ func (m *Manager) getLocalIPFS() (ipfsutil.ExtendedCoreAPI, *ipfs_core.IpfsNode,
 		// Setup MC
 		if m.Node.Protocol.MultipeerConnectivity {
 			swarmAddrs = append(swarmAddrs, mc.DefaultAddr)
-			mcOpt := libp2p.Transport(proximity.NewTransport(m.ctx, logger, mc.NewDriver(logger)))
-			p2pOpts = libp2p.ChainOptions(p2pOpts, mcOpt)
+			p2pOpts = libp2p.ChainOptions(p2pOpts,
+				libp2p.Transport(proximity.NewTransport(m.ctx, logger, mc.NewDriver(logger))),
+			)
+		}
+
+		if m.Node.Protocol.RelayHack {
+			// Resolving addresses
+			pis, err := ipfsutil.ParseAndResolveRdvpMaddrs(m.GetContext(), m.initLogger, []string{
+				"/ip4/51.159.21.214/udp/4040/quic/p2p/QmdT7AmhhnbuwvCpa5PH1ySK9HJVB82jr3fo1bxMxBPW6p",
+				"/ip4/51.15.25.224/udp/4040/quic/p2p/12D3KooWHhDBv6DJJ4XDWjzEXq6sVNEs6VuxsV1WyBBEhPENHzcZ",
+				"/ip4/51.75.127.200/udp/4141/quic/p2p/12D3KooWPwRwwKatdy5yzRVCYPHib3fntYgbFB4nqrJPHWAqXD7z",
```

Comment:
can you add them in the config?

Reply:
> can you add them in the config?

I've tried but that also require to update bertytypes, I'm scared to break somethings in the future at rollback.

Reply:
My bad, `bertytypes` is not a protobuf generated package.